content/sensu-core/1.4/api/results.md

```diff
@@ -351,4 +395,5 @@ Server: thin
 [3]:  https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html
 [4]:  ../../reference/clients#proxy-clients
 [5]:  ../../reference/checks#check-definition-specification
-[6]:  ../overview#pagination
\ No newline at end of file
+[6]:  ../overview#pagination
```

Comment:
I don't think there are any links to `../overview#pagination` elsewhere in the doc (although I guess it doesn't hurt to leave it)

Reply:
I see this link is being used on lines 83 and 268, so I'll leave it in place.

Reply:
I'm sorry - I must have been looking at the wrong page.